math/module_argument_cplexnum.cpp

```diff
@@ -0,0 +1,84 @@
+/**
+ * Copyright 2020 @author PaulinaQuintero
+ * @file
+ *
+ * \brief Program to calculate the modulus and argument(radians) 
+ * value for a complex number
+ * the form of a complex number be like z=x+iy
+ * so for a pair of numbers (a,b) that a represents x and b y
+ * the program can calculate the modulus like
+ * |z|=sqrt(a^2+b^2)
```

Comment:
```suggestion
 * \f$|z|=\sqrt{a^2+b^2}\f$
```
This will generate better documentation like [here](https://kvedala.github.io/C-Plus-Plus/db/dd3/ode__forward__euler_8cpp.html)